src/graphql-types/ReasonApolloQuery.re

```diff
@@ -100,14 +100,11 @@ module Get = (Config: ReasonApolloTypes.Config) => {
             "pollInterval": pollInterval |> fromOption,
             "notifyOnNetworkStatusChange":
               notifyOnNetworkStatusChange
-              |> Js.Option.map((. b) => Js.Boolean.to_js_boolean(b))
+              |> Js.Option.map((. b) => b)
```

Comment:
I think this row can now be removed. /cc @Gregoirevda ?

Reply:
wow. That is not in my version. I had remembered removing it and its not there. 

I guess I don't understand the request here. Clarification? 

Thank you!

Reply:
Yes, correct (but I'd like @Gregoirevda to confirm it) 😇

Reply:
removed

Reply:
Adding ssrMode to `Client.instance` like so, compiles without error:
```
let instance =
  ReasonApollo.createApolloClient(
    ~link=httpLink,
    ~cache=inMemoryCache,
    ~ssrMode=true,
    (),
  );
```

I dont know if this means it worked.

Reply:
Yes correct, since we don't need to cast it anymore, we don't need to map on it